.gitlab-ci.yml

```diff
@@ -64,6 +64,11 @@ opam-lint:4.11.2:
   variables:
     COMPILER: "4.11.2"
 
+opam-lint:4.13.1:
+  extends: .opam-lint
+  variables:
+    COMPILER: "4.13.1"
+
```

Comment:
In my opinion, we are just wasting energy with all these `opam-lint` jobs that do exactly the same thing. We should have only one.

Reply:
OK yes.